### PR TITLE
Introduce GliaCoreDependency framework and podspec for this.

### PR DESCRIPTION
**Pull request by dukhovnyi**

### CODE REVIEW

File: GliaCoreDependency/1.0/GliaCoreDependency.podspec [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Pod::Spec.new do |s|
+  s.name         = 'GliaCoreDependency'
+  s.version      = '1.0'
+  s.summary      = 'Glia Core SDK dependency'
+  s.description  = 'The Glia Core SDK brings the in-person customer experience to iOS devices.'
+  s.homepage     = 'https://github.com/salemove/ios-bundle'
+  s.license      = { :type => "MIT", :file => "LICENSE.md" }
+  s.author       = "ReactiveCocoa"
```
Comment (igorkravchenko) [inline GliaCoreDependency/1.0/GliaCoreDependency.podspec]: Is this correct name for `author`?

Comment (dukhovnyi) [reply GliaCoreDependency/1.0/GliaCoreDependency.podspec]: good catch. fixed - d84996a8473654a2fadacb1134cd898bf0294297

**Comment by dukhovnyi** [comment]
!fixup

**Comment by dukhovnyi** [comment]
!squash

**Review by igorkravchenko — APPROVED** [review]

**Pull request merged**